src/sdp/DefaultSDP.ts

```diff
@@ -267,6 +267,56 @@ export default class DefaultSDP implements SDP {
     return new DefaultSDP(originalSdp);
   }
 
+  getIdForAllocationRtpHeaderExtension(): number {
+    const srcLines: string[] = this.lines();
+    const headerExtensionIds: number[] = [];
+
+    for (const line of srcLines) {
```

Comment:
this seems like it needs to account for audio vs video sections?

Reply:
I think this is fine for now.  There should still be plenty of unique IDs to borrow.